### DISCUSSION TITLE
Problems with german transcription

### DISCUSSION THREAD

**Discussion by magic-tech-dev** [deepgram/community#138] [General help] [answered]
I'm experiencing problems with **German Live transcription**. Both opponents are speaking German (native German speakers), Deepgram configured like this:

```
{
  punctuate: true,
  interim_results: false,
  language: 'de',
  tier: 'enhanced',
  model: 'general',
  smart_format: false,
  timeSlice: 250,
}
```

When it is silence inside the stream Deepgram just sending words **Wir** or **Ich** in final messages. 
I thought maybe it is something sent somehow in the media stream but we also tried to completely mute everybody and it is still reproducible. Also did recording of the media stream and didn't find any sound in that timings.
Same config (with changed to 'en' language option) is working great for English language.

Tried to play a bit with **timeSlice** (increased it to 500) but this didn't really help. Not sure if **endpointing** option might help.

Is it possible to somehow prevent receiving **Wir** or **Ich** in final messages?

**Comment by shirgoldbird** [comment]
Hi @magic-tech-dev! Sorry you ran into this. Are you using a browser microphone for transcription? Sometimes that can introduce artifacts that our models incorrectly transcribe as words. Testing out different audio sources may help; I'll also talk with our research team to see if there's anything we can do on our end.

I'm not aware of a `timeSlice` option in our API. Did you find that parameter documented somewhere? `endpointing` is designed to send back transcripts as soon as speech is complete, so it's unlikely to help in this case.

**Reply by 3adel** [reply]
Facing the same weird issue with "Wir" and "Ich"

**Comment by magic-tech-dev** [comment]
Hi @shirgoldbird! Yes, I'm using browser microphone for transcription. Our product is a web client, so using other resources is not possible unfortunately. 

`timeSlice` is not part of your API, this is part of my recorder implementation. It reflects how often the data will be sent to Deepgram.
 
About `endpointing` it is written that "Deepgram’s Endpointing feature monitors incoming streaming audio and detects sufficiently long pauses that are likely to represent an endpoint in speech." - I just thought maybe it can better detect pauses in speech. The pause is just the presence (absence) of sound below a certain level of dB and if you're talking about artifacts then using this option indeed doesn't make sense.

> Sometimes that can introduce artifacts that our models incorrectly transcribe as words.

But why then it is working correct with 'en' language? I'm using the same config and microphone - never saw something like that. English model somehow ignores this artifacts?

English transcription working great. Seems like the German recognition is not production ready - written POC is not working correctly. Please let me know if it will be any workaround available to make it working more stable in case of German language recognition.

**Reply by PetabitMan** [reply]
I would recommend using the whisper model instead works fine for me with german transcriptions.

**Reply by magic-tech-dev** [reply]
@PetabitMan If you mean Deepgram Whisper - based on the documentation it is not supporting LIVE transcription. You can find documentation  [here](https://developers.deepgram.com/documentation/guides/deepgram-whisper-cloud/).

**Reply by shirgoldbird** [reply, marked as answer]
@magic-tech-dev That's correct, Whisper does not support real-time transcription.

It seems like our German model has a recurring issue where it detects words in silence. This is due to the data the model was trained on and can't be fixed on your end by configuring Deepgram's parameters.

 I've passed along your feedback to the research team so we can investigate improvements. Unfortunately, I don't have a timeline for you at the moment, but we usually announce language model updates on our [changelog](https://deepgram.com/changelog/) and [social media ](https://twitter.com/deepgramAI).

Please let me know if you have any other questions!
